addons/binding/org.openhab.binding.yamahareceiver/OSGI-INF/org.openhab.binding.yamahareceiver.discovery.YamahaDiscoveryParticipant.xml

```diff
@@ -1,17 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-
-    Copyright (c) 2010-2017 by the respective copyright holders.
-
-    All rights reserved. This program and the accompanying materials
-    are made available under the terms of the Eclipse Public License v1.0
-    which accompanies this distribution, and is available at
-    http://www.eclipse.org/legal/epl-v10.html
-
--->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.binding.yamahareceiver.discovery">
-   <implementation class="org.openhab.binding.yamahareceiver.discovery.YamahaDiscoveryParticipant"/>
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.binding.yamahareceiver.discovery.YamahaDiscoveryParticipant">
```

Comment:
@zarusz Can you add a .gitignore file in the yamaha binding directory with the context "OSGI-INF/*.xml"?